Resources/Private/Templates/Administration/Index.html

```diff
@@ -46,7 +46,7 @@ <h1>
 	<f:if condition="{showSearchForm}">
 		<f:form id="administrationForm" name="demand" object="{demand}" method="get" data="{autoSubmitForm:autoSubmitForm}">
 			<input type="hidden" name="formSubmitted" value="1">
-            <input type="hidden" name="route" value="/module/web/NewsAdministration">
+            <input type="hidden" name="route" value="/module/web/NewsAdministration/">
```

Comment:
the slash is too much

Reply:
Yep, just tested it with 9 LTS and 10 LTS. It is needed with 9 and breaks with 10.

Reply:
Looks like the version check is unfortunately needed after all.

Reply:
I'll move my fork to a different branch to avoid the commit spam and report back when everything is tested and working with both 9 and 10.

Reply:
The slash is assumed by the older version in https://review.typo3.org/c/Packages/TYPO3.CMS/+/60433/5/typo3/sysext/core/Classes/Utility/ExtensionManagementUtility.php#b816. I adjusted the manual module path to include the slash. After the correct TCA was loaded without cache it now works with 9 LTS and 10 LTS.